libraries/RC_Channel/RC_Channels.cpp

```diff
@@ -94,6 +96,14 @@ const AP_Param::GroupInfo RC_Channels::var_info[] = {
     // @Group: 16_
     // @Path: RC_Channel.cpp
     AP_SUBGROUPINFO(obj_channels[15], "16_", 16, RC_Channels, RC_Channel),
+
+    // @Param: _OVERRIDE_TIME
+    // @DisplayName: RC override timeout
+    // @Description: Timeout after which RC overrides will no longer be used, and RC input will resume, 0 will disable RC overrides
+    // @User: Advanced
+    // @Range: 0.0 120.0
+    // @Units: s
+    AP_GROUPINFO("_OVERRIDE_TIME", 32, RC_Channels, _override_timeout, 5.0),
```

Comment:
5 seconds seems to be a big default timeout.

Reply:
Cut to 3, which was the lower floor from the dev call.